css/excercise_one/part-d/excercise_one.html

```diff
@@ -0,0 +1,351 @@
+        <img class="position-absolute left-0 top-0 reduce-size" src="../../../img/quote_vector.png" alt="&quot;">
+        <hr class="mt-lg mb-md standard-border w-100">
+        <p class="font-weight-medium text-md mb-xl letter-spacing-negative line-height-sm">
+          Muchísimas opciones concentradas en un solo lugar.  Poder comparar, elegir y gestionar todo desde la web es increíble.
+        </p>
+        <div class="d-flex">
+          <img src="../../../img/rootstrap_client.png" alt="Roostrap logo">
+          <div class="d-flex flex-column ml-md">
+            <p class="font-weight-medium text-sm m-0">
+              Sandra De La Roche
+            </p>
+            <p class="font-weight-light text-sm m-0">
+              Administration Specialist
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="d-flex flex-column justify-content-between align-items-center px-xl">
+    <h2 class="font-weight-extra-bold text-xxxl text-center line-height-xlg dark-blue-mobile text-xxl-mobile">
+      ¿Tenés un servicio o producto para ofrecer?
+    </h2>
+    <h6 class="font-weight-medium text-md text-center m-0 line-height-sm big-screen-display">
+      Si ofreces servicios o productos para empresas, PlanIT es el lugar para vos. Escribí tus datos y nos contactaremos contigo.
+    </h6>
+    <h4 class="font-weight-bold text-md-mobile text-center m-0 line-height-sm mobile-screen-display">
+      Si ofreces servicios o productos para empresas, PlanIT es el lugar para vos. Dejanos tus datos y nos contactaremos contigo a la brevedad.
+    </h4>
+    <div>
+      <button class="standard-shadow rounded border-transparent dark-blue-gradient font-white text-sm font-weight-bold padding-buttons mt-lg text-sm-mobile">
+      QUIERO SER PARTE
+    </button>
+    </div>
+  </div>
+  <div class="d-flex flex-column tertiary-background-color mt-xxl top-rounded px-xl">
+    <div class="d-flex justify-content-between mt-xl flex-column-mobile gap-lg mt-md-mobile">
+      <h2 class="font-weight-extra-bold text-xxxl font-white line-height-xlg m-0 text-xxxl-mobile">
+        ¿Listo para organizarte <span class="tertiary-color">10 veces</span> más rápido?
+      </h2>
+      <div class="d-flex flex-column justify-content-between w-100">
+        <div class="mb-xs text-sm text-md-mobile font-weight-medium d-flex flex-column justify-content-between gap-md">
+          <input type="text" placeholder="Empresa" class="custom-input mb-xs">
+          <input type="text" placeholder="Email" class="custom-input mb-xs">
+          <input type="text" placeholder="Mensaje" class="custom-input mb-xs">
+        </div>
+        <button class="standard-shadow rounded primary-background-color border-transparent mt-lg">
+          <p class="dark-blue-gradient text-transparent background-clip-text text-sm font-weight-medium m-0 py-sm text-md-mobile">
+            ENVIAR
+          </p>
+        </button>
```

Comment:
inside the button you don't need to put tags for paragraph or heading, the classes used here can go on the button itself and It should work.